x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -127,10 +139,20 @@ const navigationTree: NavigationTreeDefinition = {
   footer: [
     {
       type: 'navGroup',
-      id: 'projest_settings_project_nav',
-      title: 'Project settings',
+      id: 'devTools',
```

Comment:
I want to check with the UX team if the dev tool was intentionally left out of the observability offering mockups or if it was an oversight.

 I don't want to hold up the work, I can revisit it later if needed.

Reply:
Ok cool thanks. Yeah you can always remove it if you don't need it 👍